lib/charms/rolling_ops/v0/rollingops.py

```diff
@@ -374,7 +374,7 @@ def _on_acquire_lock(self: CharmBase, event: ActionEvent):
             Lock(self).acquire()  # Updates relation data
             # emit relation changed event in the edge case where aquire does not
             relation = self.model.get_relation(self.name)
-            self.charm.on[self.name].relation_changed.emit(relation)
+            self.charm.on[self.name].relation_changed.emit(relation=relation, app=self.charm.app, unit=self.charm.unit)
```

Comment:
Change on a lib without version change?

Reply:
Ah, I forgot to publish it. Will fix in another PR, but I did merge it here - https://github.com/canonical/charm-rolling-ops/pull/5